ios/ViewController.mm

```diff
@@ -30,23 +31,36 @@
 #define IS_IPAD() ([UIDevice currentDevice].userInterfaceIdiom == UIUserInterfaceIdiomPad)
 #define IS_IPHONE() ([UIDevice currentDevice].userInterfaceIdiom == UIUserInterfaceIdiomPhone)
 
-class IOSDummyGraphicsContext : public DummyGraphicsContext {
+class IOSGraphicsContext : public DummyGraphicsContext {
 public:
-	IOSDummyGraphicsContext() {
+	IOSGraphicsContext() {
 		CheckGLExtensions();
 		draw_ = Draw::T3DCreateGLContext();
+        renderManager_ = (GLRenderManager *)draw_->GetNativeObject(Draw::NativeObject::RENDER_MANAGER);
```

Comment:
Something went wrong with indentation here, maybe it's wrong in the code but it should be consistent at least :)